soes/include/sys/gcc/cc.h

```diff
@@ -8,8 +8,13 @@ extern "C"
 
 #include <assert.h>
 #include <stdint.h>
+#include <stddef.h>
 #ifdef __linux__
    #include <endian.h>
+   #ifndef MAX
+      #define MIN(a,b) (((a)<(b))?(a):(b))
+      #define MAX(a,b) (((a)>(b))?(a):(b))
+   #endif
```

Comment:
These MIN/MAX macros are in sys/param.h, although it doesn't seem like they are used for the linux build?

Reply:
They are used for Linux, I'll add sys/param.h

#include <assert.h>
#include <stdint.h>
#include <stddef.h>
#include <sys/param.h>